llm_assistant/discord_bot/ai/qa.py

```diff
@@ -0,0 +1,36 @@
+import os
+import dotenv
+from openai import OpenAI, AsyncOpenAI
+
+import time
+
+dotenv.load_dotenv()
+
+
+async def async_answer_question(model, question):
+  start = time.time()
+  ai_server = os.environ.get("ai_server", "http://localhost:8000")
+  client = AsyncOpenAI(base_url=ai_server, api_key="FAKE")
+  response = await client.chat.completions.create(
+    model=model,
+    messages=[{"role": "user", "content": question}],
+  )
+  out = response.choices[0].message.content
+
+  return out, time.time() - start
+
+
+def answer_question(model, question):
```

Comment:
Do we use this function anywhere else, besides the `__main__` ? if yes, might as well move it inside the main. :-/

Reply:
yeah that make sense.